src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,188 @@
+deployment. All the other settings like service discovery enablement and globalnet support are sourced from the
+broker-info file.
+
+
+##### join flags (general)
+
+    -h, --help                           help for join
+
+    --cable-driver string                Cable driver implementation
+                                         (defaults to strongswan -IPSec-)
+
+    --clusterid string                   Cluster ID used to identify the tunnels
+                                         Every cluster needs to have an unique cluster ID, if not provided
+                                         subctl will prompt the admin for a cluster ID.
+
+    --clustercidr string                 Cluster CIDR
+                                         specify a cluster CIDR (generally the IP addresses used for the pods in this 
+                                         cluster), if not specified subctl will try to discover this detail, and if
+                                         it's unable to discover it will prompt the admin.
+
+    --colorcodes string                  color codes (default "blue"), coma separated values
+                                         please use with caution, as it may change in the future. This setting is
+                                         used to make submariner gateway form connections only with clusters
+                                         which share the same color code. This will be replaced by something
+                                         more flexible in the future.
+
+    --no-label                           skip gateway labeling
+                                         this will disable prompting the administrator for a worker node to use
+                                         as gateway.
+
+    --subm-debug                         Enable Submariner debugging (verbose logging)
+
+    --disable-cvo                        disable OpenShift's cluster version operator if necessary, without prompting
+                                         currently, lighthouse service modifies the cluster dns operator in OpenShift
+                                         and we need to disable CVO, this setting disables the prompt which warns the
+                                         administrator.
+
+##### join flags (globalnet)
+
+    --globalnet-cluster-size uint        Cluster size for GlobalCIDR allocated to this cluster (amount of global IPs).
+
+    --globalnet-cidr string              GlobalCIDR to be allocated to the cluster, this setting is exclusive with
+                                         --globalnet-cluster-size and configures a specific globalnet CIDR for this
+                                         cluster.
+
+##### join flags (IPSec)
+
+    --disable-nat                        Disable NAT for IPsec.
+
+    --ikeport int                        IPsec IKE port (default 500)
```

Comment:
```suggestion
    --ikeport <value>                        IPsec IKE port (default 500)
```

I suggest using **\<value\>** instead of **int** or **uint** in all places.